### ISSUE TITLE
Note: This fix no longer works

### ISSUE THREAD

**Issue by szhu** [szhu/fix-growlvoice#12]
A few days ago, Google pulled the ClientLogin API, which means that the app will no longer be able to sign in on startup. This issue is beyond the scope of this repo, which only fixed a JSON bug, and this will not be fixed.

**Comment by ScootRay** [comment]
I thought I'd load up GrowlVoice today and for some reason it started working again... Big surprise! Wonder if it works for others again now?

**Comment by szhu** [comment]
You're right! Confirmed by d235j/GVFixer#7. Closing this issue then.

**Comment by szhu** [comment]
It's back.

**Comment by jcg3** [comment]
I'm confused - GrowlVoice was working for me (thanks to this glorious github project), after updating to the latest version of OS X, it wasn't, I tried re-applying the patch... 404 from Google. Is that what everyone else is experiencing?

Was it just coincidence that the OS X update happened around the same time that Google pulled it's ClientLogin APIs again?

Thanks in advance - you made GrowlVoice usable again, even if it was temporary.

**Comment by szhu** [comment]
It worked around June 22, for about a day. If you hadn't restarted your computer since then, then that might be why.

**Comment by ylluminate** [comment]
Any options anyone has found that are similar to GrowlVoice for macOS since this sad day? I've still not found any good options and am thinking about terminating my Google Voice account since I can't get my voicemail and alerts thereof instantly now in OS...

**Comment by szhu** [comment]
Google has since allowed using Google Voice through Hangouts -- you can call, check voicemail, and text in Hangouts. You get instant notifications on mobile butt's a web app so it's not as seamless on desktop,

**Comment by szhu** [comment]
Google has since allowed using Google Voice through Hangouts -- you can call, check voicemail, and text in Hangouts. You get instant notifications on mobile but on desktop its's a web app so it's not as seamless. But unless you use 100% Apple devices and use iMessages there's nothing currently similar, I think.

**Comment by szhu** [comment]
Google has since allowed using Google Voice through Hangouts -- you can call, check voicemail, and text in Hangouts. You get instant notifications on mobile but on desktop its's a web app so it's not as seamless. But unless you use 100% Apple devices and use iMessages there's nothing currently similar, I think.

**Comment by GaryHammer** [comment]
For those of us who *are* using 100% Apple devices, what are you referring to as a solution to the Google Voice problem thru iMessages?

**Comment by ylluminate** [comment]
I too would like to know of a solution for those of us using 100%  devices. For example, I've still not found an option that facilitates transcription, downloading, etc. of voice mail like Google Voice... Would love to replace it with something equivalent!

**Comment by szhu** [comment]
> not found any good options

> what are you referring to as a solution to the Google Voice problem thru iMessages?

> Would love to replace it with something equivalent!

Can someone clarify what is lacking with Google Voice? Hangouts has decent support and all platforms.

Is the lack of a native desktop app? That's somewhat of an issue for me, but Hangouts is far from the only app that lacks a native desktop app. There's an app called [Rambox](http://rambox.pro/) specifically designed to host messaging web apps. I leave Hangouts open as a app tab in Rambox, along with some other apps as well:

![image](https://cloud.githubusercontent.com/assets/1570168/21156282/2a726e80-c12a-11e6-853f-d9ca9f3d7514.png)

Keep in mind that Apple is probably able to offer iMessages as a native app because they only have to develop for platforms that they have full control over. Supporting every platform natively is not easy for anyone!

**Comment by ylluminate** [comment]
I want a simple voice mailbox that lives in my menubar and alerts me to when there are messages and allows me to read and listen to them right there.  That's all I want from Google Voice.  I don't want Hangouts and I dislike it very much.  I use Franz for my many and varied messengers, but Hangouts has always been terrible and I don't see any hope of Google getting it's UI/UX act together anytime soon.  It just doesn't work like I want for voicemail.  So yeah, the main thing for me is a 1-to-1 for what GrowlVoice did for Google Voice.

**Comment by szhu** [comment]
I'd be hard-pressed to find any menubar apps (messaging or otherwise) with significant popup functionality these days... iMessages doesn't do that either. If you do find any, update us here!

**Comment by gkendrick** [comment]
I use the Google Voice chrome extension (https://chrome.google.com/webstore/detail/google-voice-by-google/kcnhkahnjcbndmmehfkdnkjomaanaooo?hl=en-US) from which you can text, make calls, see voice mail transcriptions and listen to voice mails. Also has icon notification when new voicemails arrive. It doesn't have all of the functionality of growlvoice, but it's the next best thing I've found. I'm a long-time google voice user.

**Comment by ylluminate** [comment]
@gkendrick yeah, have tried that here, but don't use Chrome much except for Epichrome SSBs.
